packages/tooling/fast-tooling-react/src/form/custom-controls/control.css.md

```diff
@@ -0,0 +1,140 @@
+# Abstract
+
+The CSS controls consist of an abstract control and two extended controls, inline CSS and internal CSS. These controls will provide a way to interpret CSS strings for inline and internal CSS into form UI and back to strings.
```

Comment:
By "internal" CSS, does that refer to editing the adopted style sheets of a component? I wasn't sure what that referred to.

Reply:
Oh, so, CSS can exist in 3 locations as defined by [mdn](https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps/How_CSS_is_structured) and reference to a stylesheet in the `<head>` of the DOM for example is "external", in a `<style>` element as "internal" and as the style property on an element as "inline". I've used the word "internal" to mean in the `<style>` element, I know this isn't super common nomenclature but there doesn't seem to be a better description.

Reply:
Since "external" and "internal" would functionally be the same syntax, I could change this to "styles" and "stylesheet".

Reply:
I like that distinction as it will also cover adopted stylesheets.